src/delphi/vcl.castlecontrol.pas

```diff
@@ -109,21 +109,25 @@       TContainer = class(TCastleContainerEasy)
     property OnMouseMove;
     property OnMouseWheel;
   end;
-
+{ Designtime Only }
```

Comment:
Remove the sections that you moved to other (design-time only) unit, do not only comment them out.